app/src/main/java/com/example/playlistmaker/ui/settings/activity/SettingsActivity.kt

```diff
@@ -54,5 +49,14 @@ class SettingsActivity : AppCompatActivity() {
         terms.setOnClickListener{
             vm.terms()
         }
+
+
+        vm.themeSettingsLiveData.observe(this, { themeSettings ->
+            themeSwitcher.isChecked = when (themeSettings) {
+                is ThemeSettings.LightTheme -> false
+                is ThemeSettings.DarkTheme -> true
+            }
```

Comment:
```suggestion
            themeSwitcher.isChecked = themeSettings is ThemeSettings.DarkTheme
```
Так будет проще читать